packages/core/src/DocumentsAndDatasets/index.js

```diff
@@ -0,0 +1,165 @@
+
+import Content from './Content';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  featuredDiv: {
+    display: 'block',
+    [theme.breakpoints.up('md')]: {
+      position: 'absolute',
+      top: '10%',
+      width: '28%'
+    }
+  },
+  featuredTitle: {
+    width: '100%',
+    fontSize: '2rem'
+  },
+  featuredDescription: {
+    width: '100%',
+    marginTop: '1rem'
+  },
+  documentData: {
+    backgroundColor: '#0050FF',
+    paddingLeft: '1.25rem',
+    paddingRight: '1.25rem',
+    [theme.breakpoints.up('md')]: {
+      paddingTop: '14.9rem',
+      paddingLeft: '0.5rem',
+      marginTop: 0
+    }
+  },
+  datasetData: {
+    paddingLeft: '1.25rem',
+    paddingRight: '1.25rem',
+    [theme.breakpoints.up('md')]: {
+      paddingTop: '14.9rem',
+      paddingLeft: '0.5rem',
+      marginTop: 0
+    }
+  },
+  imageHighlight: {
+    width: '100%',
+    paddingLeft: '1.25rem',
+    paddingRight: '1.25rem',
+    backgroundColor: '#0050FF',
+    [theme.breakpoints.up('md')]: {
+      width: '60%',
+      paddingLeft: 0,
+      paddingRight: 0
+    }
+  }
+}));
+
+function DocumentsAndDatasets({
+  highlightChildren,
+  title,
+  description,
+  datasetContent,
+  documentContent,
+  ...props
+}) {
+  const classes = useStyles(props);
+
+  return (
+      <Grid container className={classes.root}>
+        <Grid
+            item
+            md={5}
+            sm={12}
+            container
+            justify="center"
+            className={classes.imageHighlight}
+        >
+            {highlightChildren}
+        </Grid>
+        <Grid item sm={12} md={3} lg={3} className={classes.documentData}>
+          { (title || description) && 
+            <div className={classes.featuredDiv}>
+                <Typography className={classes.featuredTitle}>
+                    {title}
+                </Typography>
+                <Typography variant="body2" className={classes.featuredDescription}>
+                    {description}
+                </Typography>
+            </div>
+          }
+            <Content
+                children={documentContent.children}
+                title={documentContent.title}
+                contentCount={documentContent.contentCount}
+                contentType={documentContent.contentType}
+                description={documentContent.description}
+                linkTitle={documentContent.linkTitle}
+                link={documentContent.link}
+                />
+        </Grid>
+        <Grid item sm={12} md={4} container className={classes.datasetData}>
+            <Content 
+                children={datasetContent.children}
+                title={datasetContent.title}
+                contentCount={datasetContent.contentCount}
+                contentType={datasetContent.contentType}
+                description={datasetContent.description}
+                linkTitle={datasetContent.linkTitle}
+                link={datasetContent.link}
```

Comment:
We need to be able to customize `Content` classes as well.. I don't see you _forwarding_ them here.